### ISSUE TITLE
Validator throws exception when value of Nillable strings set to null

### ISSUE THREAD

**Issue by mennorijnders** [cwmiller/broadworks-connector-net#6]
Hello,

First of all. For me this is a very useful project. Thank you!!

I noticed this issue when using the UserModifyRequest22 class.

When I try to set either extension or phoneNumber to null the Validator throws an exception.
Both have Nillable=true in de specification.

The exception is thrown because the object with value null is passed into ValidatePropertyRestrictions().
Any of the value.ToString().Length requests will fail.

Test to reproduce:

```
[Fact]
public void TestNillableValidation()
{
    var request = new UserModifyRequest22()
    {
        UserId = "john.doe@test.com",
        PhoneNumber = null
    };
    var results = Validator.Validate(request);
    Assert.True(results.Success);
}
```

Thanks,
Menno

**Comment by cwmiller** [comment]
Thank you for reporting this. I'm so sorry I somehow didn't see this for the longest time. A fix for this is now in releases 1.4.1 and 2.0.1.

**Comment by mennorijnders** [comment]
@cwmiller
Great, Thank you very much.
